libraries/apollo-gradle-plugin-external/src/main/kotlin/com/apollographql/apollo3/gradle/internal/BuildDirLayout.kt

```diff
@@ -18,6 +18,11 @@ object BuildDirLayout {
         "generated/metadata/apollo/${service.name}/metadata.json"
     )
   }
+  internal fun knownOperationsManifest(project: Project, service: Service): Provider<RegularFile> {
+    return project.layout.buildDirectory.file(
+        "generated/knownOperationsManifest/apollo/${service.name}/manifest.json"
```

Comment:
>  would it make sense to name the file `knownOperationsManifest.json` 

Works for me. I guess it'd make sense to align on different platforms. @AnthonyMDev @glasser , thoughts?

> Also is the apollo folder needed?

Not really but it's been there forever. It's important for `generated/source/apollo` because that's a path several pplugins may use but for `knownOperationsManifest` not so much but ... consistency?

Reply:
Agreed about consistency 👍 And yes could be useful to decide on a common file name.